lua/nvim-navic/init.lua

```diff
@@ -288,7 +288,8 @@ function M.is_available()
 	return vim.b.navic_client_id ~= nil
 end
 
-function M.get_location()
+function M.get_location(local_config)
+	local_config = vim.tbl_deep_extend("force", config, local_config or {})
```

Comment:
This won't work, icons are setup using string names whereas they are stored with integer indices.